styles/events/events-summary-container-interactive.scss

```diff
@@ -0,0 +1,19 @@
+.events-summary-container-interactive {
+    transform: scale(1.1);
+    z-index: 100;
```

Comment:
it's forbidden to write external position in block 
and don't use 100
use 1 or 2
you should be carefully with z-index